floris/simulation/turbine.py

```diff
@@ -218,10 +213,7 @@ def power(
     for turb_type in turb_types:
         # Using a masked array, apply the thrust coefficient for all turbines of the current
         # type to the main thrust coefficient array
-        p += (
-            power_interp[turb_type](rotor_effective_velocities)
-            * (turbine_type_map == turb_type)
-        )
+        p += power_interp[turb_type](rotor_effective_velocities) * (turbine_type_map == turb_type)
 
     return p * ref_density_cp_ct
```

Comment:
Not for this PR, but for future reference, I think we'll need to change this according to @paulf81 's concept of "sliding" along the power curve.